nexus/celery.py

```diff
@@ -37,4 +41,9 @@ def on_configure(self):
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
 
-app.conf.beat_schedule = {}
+app.conf.beat_schedule = {
+    'publish-posts-task': {
+        'task': 'publish_posts_task',
+        'schedule': CELERY_BEAT_MINUTES['publish_posts_task'] * 60,
```

Comment:
In `schedule` key use: `timezone.timedelta(seconds=1800)`

Also, in settings, let's use settings instead of minutes and then converting minutes to settings to pass to conf.

Let's make it consistent with seconds.